notebook/templates/notebook.html

```diff
@@ -32,6 +32,7 @@
 data-ws-url="{{ws_url | urlencode}}"
 data-notebook-name="{{notebook_name | urlencode}}"
 data-notebook-path="{{notebook_path | urlencode}}"
+data-server-root="{{server_root}}"
```

Comment:
I would like to try that without exposing `server_root` it is potentially sensitive informations, and we tried out best to not expose it anywhere in the UI. The root is usually represented as the Home icon. 

As far as I can tell having this information is only for display purpose, so I would just stay with the home icon.

Reply:
I actually disagree. I don't think `server_root` is really that sensitive, and it's probably not hard to work out from the list of files, which is already accessible. The last time we thought about this was before we enabled authentication by default; now that we've done that, I think we can be slightly more relaxed about showing this kind of information.

We've been using the home icon for lack of anything better, but I don't think it's great UI:

- It's easy to confuse with your home directory, especially as the two are sometimes but not always the same. File managers frequently use a very similar icon for your home directory (e.g. in Gnome: 
![screenshot from 2018-04-21 08-39-43](https://user-images.githubusercontent.com/327925/39081330-a425c7fa-453f-11e8-9909-284e1a64ab7d.png)
- It obscures where the files are actually stored, if you want to go and work with them outside the notebook. At one point Anaconda's GUI launcher would create a directory like 'Notebooks' to run the server in, which is hard to discover from inside.

We should make sure it comes from the contents manager and works if it's empty, to allow for non-filesystem contents managers. But I think the benefits of showing it outweigh the disadvantages.